blog/articles/article_detailed/article_visual_debugger_in_vscode/debugger.md

```diff
@@ -212,57 +245,64 @@ with small fixes to launch.json, we have it adapted for fastapi now
 
 ![]({{.StaticRoot}}article_visual_debugger_in_vscode/fastapi1.png)
 
-With written small code examples in app_test.py we have no trouble launching visual debug for unit tests of fastapi
+With written small code examples in `app_test.py` we have no trouble launching visual debug for unit tests of fastapi
 
 ![]({{.StaticRoot}}article_visual_debugger_in_vscode/fastapi2.png)
 
 See folder `flask_example` for almost same example to try for Flask too.
 
 # Python in already existing Docker
 
-Ensure having installed Dev Containers app @id:ms-vscode-remote.remote-containers
+Ensure having installed Dev Containers app `@id:ms-vscode-remote.remote-containers`
 ![]({{.StaticRoot}}article_visual_debugger_in_vscode/docker_extension_install1.png)
 
-- go to examples/django_example folder and run `docker compose build` and `docker compose run --service-ports shell`
-    - `--service-ports` option ensures we forwarded 8000 port written in docker-compose in case of wishing to runserver and seeing it
-- check the settings configured to make things working in `docker-compose.yml`.
-    - We forwarded current volume insode to path /code, we made sure working directory in container /code too.
+```shell
+cd ../examples/django_example
+docker compose build
+docker compose run --service-ports shell
```

Comment:
```suggestion
cd ../examples/django_example
docker compose build
docker compose run --service-ports shell # `--service-ports` option ensures we forwarded 8000 port written in docker-compose despite using `run` command instead of `up`
```